zipkin-query-service/src/main/scala/com/twitter/zipkin/builder/QueryServiceBuilder.scala

```diff
@@ -37,24 +32,10 @@ case class QueryServiceBuilder(
     thriftscala.Adjust.TimeSkew -> new TimeSkewAdjuster()
   )
 
-  def addServerSetPath(p: (ZooKeeperClientBuilder, String)) = copy(serverSetPaths = serverSetPaths :+ p)
-
   def apply(): (RuntimeEnvironment) => ListeningServer = (runtime: RuntimeEnvironment) => {
-    val log = Logger.get()
     serverBuilder.apply().apply(runtime)
-
-    val address = new InetSocketAddress(serverBuilder.serverAddress, serverBuilder.serverPort)
     val store = storeBuilder.apply()
 
-    /* Register server sets */
-    serverSetPaths foreach { case (zkClientBuilder, path) =>
```

Comment:
we shouldn't take on random responsibilities such as registering arbitrary services.